website/project/views/mailing_list.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from website.project.decorators import (
+    must_be_valid_project,
+    must_have_permission,
+    must_not_be_registration,
+)
+from website.util.permissions import ADMIN
+
+from website.project.mailing_list import route_message
+
+
+###############################################################################
+# View Functions
+###############################################################################
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def get_node_mailing_list(node, auth, **kwargs):
+    return format_node_data_recursive([node], auth.user)
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def enable_mailing_list(node, **kwargs):
+    node.mailing_enabled = True
+    node.save()
```

Comment:
All views except for those for `DELETE` requests should return useful response to the user. Even something as simple as `{'message': 'Successfully enabled mailing lists', 'success': True}`.